src/main/java/com/alkemy/ong/model/Role.java

```diff
@@ -0,0 +1,40 @@
+package com.alkemy.ong.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import java.time.LocalDate;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "roles")
+public class Role {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotBlank
+    @Column(nullable = false)
```

Comment:
Se me pasó en la corrección anterior, faltan los mensajes de error, ejemplo: @NotNull(message = "name can't be null")